modules/bot.py

```diff
@@ -215,7 +261,12 @@ def _teams_picking_to_str(self):
         if self.ranked:
             unpicked_str = "[" + ", ".join(["`{0}{1}`".format(utils.rating_to_icon(self.ranks[i.id]), (i.nick or i.name).replace("`","")) for i in sorted(self.unpicked, key=lambda p: self.ranks[p.id], reverse=True)]) + "]"
         else:
-            unpicked_str = "[" + ", ".join(["`{0}`".format((i.nick or i.name).replace("`","")) for i in self.unpicked]) + "]"
+            player_strs = []
+            for position, player in self.unpicked_pool.all.items():
```

Comment:
lol turned this into a for loop instead of list comprehension. list comprehensions that involve multiple variables or a lot of string magic are super annoying and hard to read compared to a for loop.